hack/e2e.sh

```diff
@@ -23,12 +23,10 @@ oc create secret generic ci-pull-secret --from-file=.dockercfg=/tmp/cluster/pull
 oc secrets link default ci-pull-secret --for=pull
 
 # Apply oc template
-oc new-app -f dist/openshift/cincinnati.yaml \
+oc new-app -f ../template/cincinnati.yaml \
```

Comment:
It's not clear to me why this path is relative to `hack/`

Reply:
This is set by `WORKDIR` in dockerfile:
```
WORKDIR "${HOME}/cincinnati"
...
COPY --from=builder hack/e2e.sh hack/
COPY --from=builder dist/openshift/cincinnati.yaml template/
```

Reply:
Ah, I was wrong - `hack/e2e.sh` would not change pwd to `hack`. Fixed